web/pdf_viewer.js

```diff
@@ -1020,7 +1029,7 @@ class PDFViewer {
     this.update();
   }
 
-  #scrollIntoView(pageView, pageSpot = null) {
+  #scrollIntoView(pageView, timeout = null, pageSpot = null) {
```

Comment:
All of these scroll-related changes makes me a fair bit worried, since delaying scrolling feels like something that could *potentially* cause other unforeseen issues.
Note that much of this code is quite old and there's lots of edge-cases to consider, so lots more comments explaining the logic would seem like a necessity here.

Reply:
Ok, I'll see later how to improve that stuff.